src/corehost/cli/deps_resolver.cpp

```diff
@@ -497,6 +460,25 @@ bool deps_resolver_t::resolve_tpa_list(
     return true;
 }
 
+bool deps_resolver_t::check_if_entry_known(const deps_entry_t& entry, pal::string_t& path)
```

Comment:
You are not using the return value, so it will be helpful to make the return type as void and also update the method name to reflect the intent (e.g. InitKnownPackageData).